server.js

```diff
@@ -32,6 +32,8 @@ const sentryEnrichScope = require('./common/middleware/sentry-enrich-scope')
 const sentryRequestId = require('./common/middleware/sentry-request-id')
```

Comment:
Completely not the scope of this PR, but with 205 lines of code in `server.js` I wonder if we should start writing some tests for it - and it is separated out from the running of the server, which makes this a lot easier to do as well